scripts/experiments_tabular/run_parallel.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 # this runs parallel experiments over all datasets
-MODELSCRIPT=$1
-MAX_SEED=$2
-cat datasets_tabular.txt | xargs -n 1 -P 40 ./${MODELSCRIPT}.sh $MAX_SEED
\ No newline at end of file
+MODEL=$1
+NUM_SAMPLES=$2
+MAX_SEED=$3
+
+while read d; do
+    sbatch --array=1-$NUM_SAMPLES -D /home/francja5/Projects/modules/GenerativeAD/scripts/experiments_tabular/ ./${MODEL}_run.sh $MAX_SEED $d
```

Comment:
Do the 100 different experiments happen here? Does that mean they are trained in parallel? Then the check_params function is not gonna work, since it can only tell that a model was already trained if it was already saved.

Reply:
This should schedule an array job (repeating the same command with the same arguments) for each dataset. [AFAIK](https://slurm.schedmd.com/job_array.html) those task in the array are run in parallel, however we can limit the number of concurrent jobs to 1 as in `--array=1-${NUM_SAMPLES}%1`.  In some cases the search space is large and collisions may happen rarely (assuming 100 samples), however I see your point for example with `kNN`, which has really only one parameter. 
What about adding the number of concurrent as a parameter to the script?

Reply:
That sounds reasonable, the repetition of parameters is probably only gonna happen for simple models where training time is probably not going to be critical. So introducing a new script parameter sounds good.

Reply:
Added.